crates/holoom_recipe/coordinator/src/lib.rs

```diff
@@ -0,0 +1,48 @@
+pub mod holoom_recipe;
+use hdk::prelude::*;
+use holoom_types::{LocalHoloomSignal, RemoteHoloomSignal};
+use shared_utils::get_authority_agent;
+
+#[hdk_extern]
+pub fn init(_: ()) -> ExternResult<InitCallbackResult> {
+    let authority_agent = get_authority_agent()?;
+    let my_pubkey = agent_info()?.agent_initial_pubkey;
+    let mut functions = BTreeSet::new();
+    let zome_name = zome_info()?.name;
+    if my_pubkey == authority_agent {
+    }
+    functions.insert((zome_name, "recv_remote_signal".into()));
```

Comment:
not needed for recipes

Reply:
yes that is redundant and can be removed